lib/module.js

```diff
@@ -33,6 +33,9 @@ function axiosModule(_moduleOptions) {
   // Default prefix
   const prefix = process.env.API_PREFIX || moduleOptions.prefix || '/'
 
+  // HTTPS
+  const https = !!(this.options.server && this.options.server.https)
```

Comment:
I'd prefer ```suggestion
  const https = Boolean(this.options.server && this.options.server.https)
```

A few more characters but more declarative/speaking

Reply:
Looks good to me